test/client/api.test.js

```diff
@@ -0,0 +1,139 @@
+/* @flow */
```

Comment:
are you open to moving these right next to the implementation? so:

```
src/
  api.js
  api.test.js
```

instead of:

```
src/
  api.js
test/
  api.test.js
```

just for consistency with paypal-checkout-components and smart-payment-buttons?

Reply:
I really like colocated tests. Would be potentially easier to consider this an incremental change (fast follow) for another PR pending agreement on this one so far. But would be cool with seeing it all in one PR too, whichever is easiest for author.

Reply:
this was our agreed upon structure moving forward: https://github.com/paypal/paypal-smart-payment-buttons/tree/main/test

Reply:
I am _absolutely_ a fan of co-locating the tests. That should be a small change so I'll timebox it for this PR.

Reply:
Thanks for suggesting this @jshawl, I've updated it accordingly!